test/utils.js

```diff
@@ -3,11 +3,25 @@
 /* global chai */
 
 describe('MasqCommon utils', () => {
+  // track created databases to be able to delete them after each tests
```

Comment:
Maybe we should wrap deleteDatabase around a promise, because the deletion of the db is async. However, I think the browser automatically handle this for us.